pandas/tests/frame/test_alter_axes.py

```diff
@@ -522,22 +522,72 @@ def test_rename_axis_inplace(self, float_frame):
     def test_rename_axis_warns(self):
         # https://github.com/pandas-dev/pandas/issues/17833
         df = DataFrame({"A": [1, 2], "B": [1, 2]})
-        with tm.assert_produces_warning(FutureWarning) as w:
+        with tm.assert_produces_warning(FutureWarning,
+                                        check_stacklevel=False) as w:
```

Comment:
So these `check_stacklevel` changes can hopefully all be reverted.

Reply:
done